lib/circlex/struct/wire_instructions.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Circlex.Struct.WireInstructions do
```

Comment:
Can we also add struct test for this module?

Reply:
fixed